File: LinkedList/easy/3515-find-if-digit-game-can-be-won/find-if-digit-game-can-be-won.java
class Solution {
    public boolean canAliceWin(int[] nums) {
        int sumOfSD=0,sumOfDD=0;

        for(int i:nums)
        {
            if(i<10)
                sumOfSD+=i;
            else
                sumOfDD+=i;
        }

        if(sumOfSD >sumOfDD ||sumOfSD <sumOfDD )
            return true;
        return false;

        
    }
}

File: LinkedList/easy/3515-find-if-digit-game-can-be-won/find-if-digit-game-can-be-won.cpp
class Solution {
public:
    bool canAliceWin(vector<int>& nums) {
        int sumOfSingleDigit=0;
        int sumOfDoubleDigit=0;
        for(auto i:nums)
        {
            if(i<10)
                sumOfSingleDigit+=i;
            else if(i>=10)
                sumOfDoubleDigit+=i;
        }
        // cout<<sumOfSingleDigit<<" "<< sumOfDoubleDigit;
        if(sumOfSingleDigit>sumOfDoubleDigit ||sumOfSingleDigit<sumOfDoubleDigit)
            return 1;
        else
            return 0;
    }
};